packages/mobile/src/send/SendConfirmation.test.tsx

```diff
@@ -83,20 +84,18 @@ describe('SendConfirmation', () => {
     )
 
     // Initial render.
+    // Await to force fees to render prior to checking snapshot and attempting press
+    await waitForElement(() => fireEvent.press(tree.getByText('feeEstimate')))
+    // TODO (anton): line 744 of the snapshot 'rotate: ...' sometimes differs for unknown reasons
     expect(tree).toMatchSnapshot()
-    fireEvent.press(tree.getByText('feeEstimate'))
```

Comment:
The tree must not be pressed before the fees are calculated (attempting to do so will cause an error because undefined fees are presented in a non-pressable element as part of this PR)